### PR TITLE
Add travis deploy

### PR DESCRIPTION
**Pull request by iagodahlem**
We still need to create a `token` for the organization, some owner needed here. And add this token on Travis as `GITHUB_TOKEN`, again owner needed here to do this.

### CODE REVIEW

**Review by paulodiovani — CHANGES_REQUESTED** [review]
Add a badge to readme too

**Review by paulodiovani — APPROVED** [review]

**Comment by matheusazzi** [comment]
@iagodahlem is it ready?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  - 2.4
+
+cache: bundler
+
+before_install:
+  - nvm install node
+
+install:
+  - bundle install --path vendor/bundle
+  - npm install
```
Comment (paulodiovani) [inline .travis.yml]: As discussed on #23, let's use Yarn.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+language: ruby
+rvm:
+  - 2.4
+
+cache: bundler
```
Comment (paulodiovani) [inline .travis.yml]: I think you can add here too:

```yml
cache:
  dirs:
    - node_modules
```

Check documentation for correct syntax

**Review by paulodiovani — CHANGES_REQUESTED** [review]

**Review by paulodiovani — APPROVED** [review]
@iagodahlem  We can also make that the `yarn.lock` was updated with `yarn check` command.

**Comment by iagodahlem** [comment]
@matheusazzi it is not ready yet, but I know what to do now to generate the token. How you are an owner from the organization, could you generate the token for me?

**Comment by iagodahlem** [comment]
@paulodiovani ok, I will add this check as well.

**Comment by iagodahlem** [comment]
Token created. Now I think that it's ready to merge.

**Pull request merged**